### ISSUE TITLE
Adjust default options [suggestion]

### ISSUE THREAD

**Issue by ootada** [tim-we/tabs-aside#107]
I thing some options might be worth to have updated defaults:

- 'New session title template': "New $y-$M-$d $H:$m"
Time based template is more useful, demonstrates whats possible and is far easier to change to plain text than the other way around if you would wish so.

- 'Set aside pinned tabs': checked
I think putting aside whole windows is more intuitive than leaving some tabs excluded as a default.

**Comment by tim-we** [comment]
Good idea, though I think the "New" seems a bit weird. What do you think about about "Session $y-$M-$d $H:$m"?

**Comment by ootada** [comment]
'Session $...' seems fine.

In my use case I always try to rename sessions to something meaningful afterwards, either leaving the date or not.
I have chosen 'New..' probably to signify that its not renamed yet and as its a short word, the full date/time can be seen not obscured.
Not sure how well this translates to others.

**Comment by tim-we** [comment]
Hmm I think shorter is better since by default the sidebar has a very small width. Perhaps "Tabs $..."?
